### PR TITLE
Fix logging format

### PR DESCRIPTION
**Pull request by mjangda**
- [x] Output timestamp in UTC (not ISO)
- [x] Don't include timestamp in JSON (breaks parsing)
- [x] Strip leading `:` from `app_type`

### CODE REVIEW

**Comment by mjangda** [comment]
@elazzabi this should fix the issues we've seen with log parsing not working. I haven't added any tests but something we should look into (may need a separate set of tests for local vs not-local log output and verify that the output matches what's expected).

File: src/logger/index.js [hunk 1]
```diff
@@ -38,6 +40,10 @@ const localLoggingFormat = printf( output => {
 // Logging format for production
 const prodLoggingFormat = printf( output => {
 	const { timestamp: time, app, app_type: type } = output;
+
+	// Can't include the timestamp in the JSON
+	delete output.timestamp;
+
 	return `${ time } ${ app }${ type } ${ JSON.stringify( output ) }`;
```
Comment (elazzabi) [inline src/logger/index.js]: given we removed the leading `:`, I think we need to add it in the logging format here (both prod and local). Right?

The current behavior is, for domain `vip:go:server` it shows `vipgo:server` in the logs

**Comment by elazzabi** [comment]
@mjangda out of curiosity, I tried to access the logs of the tests transport directly and it's working. I pushed a simplification of the tests here ad50505fd08d24c48e0c91d6a541283716636ccb. What do you think?

File: src/logger/index.js (hunk 1 above)
Comment (mjangda) [reply src/logger/index.js]: Oh good catch :)

**Comment by mjangda** [comment]
> I pushed a simplification of the tests here ad50505.

Looks great!

I pushed a fix for the missing `:` and adding some tests to help catch the issue as well.

**Comment by mjangda** [comment]
The reason is because `message` (set by us) and `symbolForMessage` (set by winston) are different keys and different things. The value for the key `symbolForMessage` is the actual output of the log entry.

**Pull request merged**